app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,42 @@
+class NotesController < ApplicationController
+
+  def index
+    # binding.pry
+    if current_user
+      @notes = current_user.notes
+    else
+      @notes = Note.all
+    end
+    # binding.pry
+    render json: @notes
+  end
+
+  def create
+    @note = Note.new(note_params)
+    if @note.save
+      render json: @note
+    else
+      error = @note.errors.full_messages.collect do |error_message|
+        {:error => error_message}
+      end
+      @errors = {:errors => error}
+      render json: @errors, status: 400
+    end
+  end
+
+  private
+
+  def note_params
+    pre_note_params = params.permit(:title,
+                             :body,
+                             :tags,
+                             :user_id)
```

Comment:
You don't want to permit user_id to be set manually ... this would allow anyone to post as any user so long as they knew the id of said user.